src/commonMain/kotlin/exchange.dydx.abacus/state/manager/StateManagerAdaptor.kt

```diff
@@ -1750,6 +1757,121 @@ open class StateManagerAdaptor(
         return null
     }
 
+    private fun triggerOrderPayload(triggerOrder: TriggerOrder, subaccountNumber: Int, marketId: String, size: Double): HumanReadablePlaceOrderPayload {
+        val clientId = Random.nextInt(0, Int.MAX_VALUE)
+        val type = triggerOrder.type?.rawValue ?: throw Exception("type is null")
+        val side = triggerOrder.side?.rawValue ?: throw Exception("side is null")
+
+        val price = triggerOrder.summary?.price ?: throw Exception("summary.price is null")
+        val triggerPrice = triggerOrder.price?.triggerPrice ?: throw Exception("triggerPrice is null")
+
+        val reduceOnly = true
```

Comment:
what's the plan for this one? do we need to feature gate stop limit / take profit limit because reduce only can't be true for GTT

Reply:
we'll be feature gating stop limit / take profit limit on the FE! (if something does end up going through, the order will just fail)